extensions/amp-story/1.0/test/test-amp-story-bookend.js

```diff
@@ -75,8 +76,11 @@ describes.realWin('amp-story-bookend', {
     storyElem = win.document.createElement('amp-story');
     storyElem.appendChild(win.document.createElement('amp-story-page'));
     win.document.body.appendChild(storyElem);
-    story = new AmpStory(storyElem);
-    bookend = new Bookend(win, story.element);
+    new AmpStory(storyElem);
```

Comment:
Just wondering: do you know why we need to create a story here?

Reply:
Woops, didn't need it anymore - it was used for a previous version of the test.